website/templates/emails/contributor_added.txt.mako

```diff
@@ -1,7 +1,13 @@
+<%!
+    from website import settings
+%>
+
 Hello ${user.fullname},
 
 ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
 
+You will automatically be subscribed to notification emails for updates to this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
Is this necessarily true? If a user sets their global default to no notifications, this line would probably be confusing. Consider passing a var to the email's template constructor, and adding logic to say either `will not be subscribed` or `will automatically be subscribed` depending on their setting.